hc/settings.py

```diff
@@ -136,7 +136,15 @@
 )
 COMPRESS_OFFLINE = True
 
+#send emails using Gmail.
 EMAIL_BACKEND = "djmail.backends.default.EmailBackend"
+DJMAIL_REAL_BACKEND = 'djmail.backends.async.EmailBackend'
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = os.environ.get("EMAIL_HOST")
+EMAIL_PORT = os.environ.get("EMAIL_PORT")
+EMAIL_HOST_USER = os.environ.get("DJMAIL_EMAIL")
+EMAIL_HOST_PASSWORD = os.environ.get("DJMAIL_PASSWORD")
+EMAIL_USE_TLS = True
```

Comment:
For these changes that require you to use environment variables, add the keys to the `.env.example` with sample values so that everyone has the same settings.